index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube Video Page</title>
+    <link rel="stylesheet" href="index.css">
+    <link href='https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css' rel='stylesheet'>
+</head>
+<body>
+    <!--------------Header---------------------------->
+    <header id="header">
+        <div id="header-container">
+            <div class="header-logo">
+                <img src="images/629733e4dc11727dae6827e7525878a05718cf48823131de8e92213c00808014_122x56.png" alt="header-img">
+            </div>
+
+            <nav id="nav-bar">
+
+                <div class="search-content">
+                    <div class="search-box">
+                        <input type="text" placeholder="Search">
+                        <i class='bx bx-search'></i>
+                    </div>
+
+                </div>
+
+
+                <div class="icon">
+                    <i class='bx bx-microphone'></i>
+                    <i class='bx bx-video-plus'></i>
+                    <i class='bx bx-bell'></i>
+                </div>
+                
+            </nav>
+        </div>
```

Comment:
Please be careful with more than one blank line and indentation you only need them for readability purpose.